assets/scripts/menus/HelpMenu.jsx

```diff
@@ -19,8 +19,10 @@ export default class HelpMenu extends React.PureComponent {
   }
 
   onClickAbout () {
-    const mountNode = document.getElementById('dialogs-react')
-    ReactDOM.render(<AboutDialog />, mountNode)
+    store.dispatch({
+      type: SHOW_DIALOG,
+      name: 'ABOUT'
```

Comment:
It's definitely not a big deal. I was suggesting putting ```export const ABOUT = 'ABOUT' ``` somewhere and then import it where you want to use it. Like Actions.